skin-schema.json

```diff
@@ -507,6 +515,14 @@
               "iOS SDK": "Unsupported",
               "Android SDK": "v4.21.0+"
             }
+          },
+          "inactiveColor" : {
```

Comment:
What is the difference between these two colors?

Reply:
Nevermind, the diff made it look like they were specified next to each other. This makes more sense if I look at the file as a whole.